kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -245,26 +312,18 @@ define([
                     title: span(['File Paths']),
                     name: `${cssClassType}s-area`,
                     body: [
-                        table(
-                            {
-                                class: `${cssBaseClass}__table`,
-                                dataElement: `${cssClassType}-fields`,
-                            },
-                            [
-                                tr({
-                                    class: `${cssBaseClass}__table_row`,
-                                    dataElement: `${cssClassType}-fields-row`,
-                                }),
-                            ]
-                        ),
+                        table({
+                            class: `${cssBaseClass}__table`,
+                            dataElement: `${cssClassType}-fields`,
+                        }),
```

Comment:
Since we aren't using any features of an HTML table, it might make more sense to use an ordered list for this -- we'd get the row numbering for free and row numbers would update automatically when an element was removed.

Reply:
That's a very good point. I mostly left it as-is for no good reason, really, when I went and changed everything else.

Reply:
Updated to use `ol` instead of `table`, and tweaked associated scss.